src/Utils/CliMsgFormatter.php

```diff
@@ -118,7 +118,7 @@ public function progressCompact( int $i, int $total, ?int $current = null, ?int
 			$last = $total;
 		}
 
-		$progress = min( 100, round( ( $i / $total ) * 100 ) );
+		$progress = !$total ? 100 : min( 100, round( ( $i / $total ) * 100 ) );
```

Comment:
I see there is a copy of this logic in the `progress` method. It'd be nice to unify that by extracting a caclulateProgress method used in both places. That new method could use a guard clause instead of this new trinry, and it could use an explicit check for $total being 0, rather than casting the ineger to boolean.

https://www.entropywins.wtf/blog/2019/01/14/readable-functions-guard-clause/
https://dev.to/jeroendedauw/readable-comparisons-2f4o

Reply:
Thanks, done.